lib/gui/components/update-notifier.js

```diff
@@ -138,17 +166,48 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_DAYS, M
    */
   this.notify = () => {
     return this.getLatestVersion().then((version) => {
-      return ModalService.open({
-        template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
-        controller: 'UpdateNotifierController as modal',
-        size: 'update-notifier',
-        resolve: {
-          options: _.constant({
-            version
-          })
+      settings.set('lastUpdateNotify', Date.now());
+      settings.set('sleepUpdateCheck', false);
```

Comment:
Oh sorry, the only thing my diff does is reduce the usage of two settings down to one. There's no bug in the current code, my diff and the current code should behave identically. I was just trying to explain to @lurch why `sleepUpdateCheck` was used, and then during this I realized it isn't necessarily needed hence the diff and all.